domino/domino.py

```diff
@@ -75,6 +76,7 @@ def runs_start(self, command, isDirect=False, commitId=None, title=None,
         }
 
         response = requests.post(url, auth=('', self._api_key), json=request)
+        raise_status_with_body(response)
```

Comment:
Please create _post wrapper method (like _get) and consolidate this (raise_status_with_body) in it